Sources/Extensions/AppIntents/Widget/Sensor/WidgetSensorsAppIntentTimelineProvider.swift

```diff
@@ -167,10 +166,11 @@ struct WidgetSensorsEntry: TimelineEntry {
     var sensorData: [SensorData] = []
 
     struct SensorData {
-        var entityId: String
+        var id: String
```

Comment:
is this the entityID or the "serverId-entityId"?

Reply:
You are right, I wanted to make it more uniform with the other widgets but forgot changing the content. I changed it so the value is filled with `sensor.id` instead of  `sensor.entityId` now